### PR TITLE
Revert get info from hass

### PR DESCRIPTION
**Pull request by bramkragten**

### CODE REVIEW

File: src-11ty/_includes/base.html [hunk 1]
```diff
@@ -53,10 +53,8 @@
       }
 
       @media all and (max-width: 500px) {
-        .layout {
-          justify-content: flex-start;
```
Comment (balloob) [inline src-11ty/_includes/base.html:57]: This keeps the card now centered on mobile. Is that what we want?

Comment (balloob) [reply src-11ty/_includes/base.html:57]: Screen Shot 2021-02-17 at 08 22 07

File: src/entrypoints/my-index.ts [hunk 2]
```diff
@@ -8,36 +8,44 @@ import {
...
-    if (this._updatingUrl) {
+    if (!this._instanceUrl || this._updatingUrl) {
       return html`
         <div class="card-content">
+          ${!this._instanceUrl
+            ? html` <h1>Setup My Home Assistant</h1>
+                <p>
+                  You have not setup My Home Assistant yet, to get started, copy
+                  the URL of your Home Assistant instance in the input below and
+                  press update.
```
Comment (balloob) [inline src/entrypoints/my-index.ts:43]:
```suggestion
                  My Home Assistant is not configured yet. Copy the URL
                  of your Home Assistant instance below and press update.
```

File: src-11ty/_includes/base.html (hunk 1 above)
Comment (bramkragten) [reply src-11ty/_includes/base.html:57]: I liked it most for the redirect at least, but your screen is way higher than mine 😄

Comment (balloob) [reply src-11ty/_includes/base.html:57]: This is just on my mac ofc, not a real phone. Ok let's keep it .

**Review by balloob — APPROVED** [review]

**Pull request merged**